content/exchange/artifacts/Windows.Forensics.Jumplists.AutomaticDestinations_JLECmd

```diff
@@ -0,0 +1,76 @@
+name: Windows.Forensics.Jumplists.AutomaticDestinations_JLECmd
+description: |
+    * Execute Eric Zimmerman's JLECmd to parse AUTOMATICDESTINATIONS-MS files in C:\ drive recursively and return output for analysis. (jlecmd.exe -d C:/ --csvf -csv tmpdir results.csv). 
+    * JLECmd.zip is downloaded from the URL to 'C:\Program Files\Velociraptor\Tools' folder.   
+    * JLECmd.zip can be uploaded to Velociraptor Server in order to copy it to the clients in case there is no internet connection.
+    * Created using @carlos_cajigas LECmd VQL as a quide.  
+    * JLECmd is a CLI tool for analyzing Custom Destinations jump list data. Learn more - https://github.com/EricZimmerman/JLECmd
+
+author: Orhan Emre @orhan_emre
+
+type: CLIENT
+
+tools:
```

Comment:
Can you add an expected hash to this artifact ?

Here is an example of another tool: https://github.com/Velocidex/velociraptor-docs/pull/789/files#diff-c34530120068a449fa566d79550de2caf5a5feb479b8392473a0b18ac20ea8a6R17

Reply:
Thank you for the feedback. I have added the SHA256 hash value and the version of the tool.

Reply:
Done!